match2/wikis/rainbow_six/match_summary.lua

```diff
@@ -0,0 +1,579 @@
+				:wikitext(score or '')
+				:css('padding','0 0 3px 3px')
+
+	self.bottom:node(roundScore)
+	return self
+end
+
+function Score:addEmptyOvertime()
+	local roundScore = mw.html.create('td'):css('width','20px')
+	self.top:node(roundScore)
+	self.bottom:node(roundScore)
+	return self
+end
+
+function Score:create()
+	self.table:node(self.top):node(self.bottom)
+	return self.root
+end
+
+local MapVeto = Class.new(
+	function(self)
+		self.root = mw.html.create('div'):addClass('bracket-popup-body-match')
+		self.table = self.root:tag('table')
+			:addClass('wikitable-striped'):addClass('collapsible'):addClass('collapsed')
+			:css('width', '100%'):css('margin-top', '-2px')
+			:css('margin-bottom','-2px'):css('border-top','1px solid #DDD')
+		self:createHeader()
+	end
+)
+
+function MapVeto:createHeader()
+	self.table:tag('tr'):css('background-color','#EEE'):css('color','#000'):css('height','10px')
+		:tag('th'):css('width','33%'):done()
+		:tag('th'):css('width','34%'):css('padding','5px'):css('font-size','11px')
+		:css('text-align','center'):wikitext('Map Veto'):done()
+		:tag('th'):css('width','33%'):done()
+	return self
+end
+
+function MapVeto:vetoStart(firstVeto)
+	local textLeft = ''
+	local textCenter = ''
+	local textRight = ''
+	if firstVeto == 1 then
+		textLeft = "'''Start Map Veto'''"
+		textCenter = '[[File:Arrow sans left.svg|15x15px|link=|Left team starts]]'
+	elseif firstVeto == 2 then
+		textCenter = '[[File:Arrow sans right.svg|15x15px|link=|Right team starts]]'
+		textRight = "'''Start Map Veto'''"
+	else return self end
+	self.table:tag('tr'):css('border-bottom','5px solid #DDD'):css('background-color','#f5f5f5')
+		:css('height','10px'):css('font-size','11px')
+		:tag('th'):css('text-align','center'):css('padding','5px'):wikitext(textLeft):done()
+		:tag('th'):css('text-align','center'):css('padding','5px'):wikitext(textCenter):done()
+		:tag('th'):css('text-align','center'):wikitext(textRight):done()
+	return self
+end
+
+function MapVeto:addDecider(map)
+	if Logic.isEmpty(map) then
+		map = 'TBD'
+	else
+		map = '[['..map..'/siege|'..map..']]'
+	end
+	self.table:tag('tr'):css('border-top','1px dotted #DDD'):css('background-color','#FFF')
+			:css('height','10px'):css('font-size','11px')
+		:tag('td'):css('text-align','center'):css('font-weight','bold')
+			:tag('span'):css('color','#000'):css('background-color',Color['stay']):css('border','none')
+				:css('border-radius','0px'):css('letter-spacing','0.1em')
+				:css('font-family','\'Source Code Pro\',monospace')
+				:wikitext('DECIDER'):done()
+			:done()
+		:tag('td'):css('text-align','center'):css('padding','5px'):wikitext(map):done()
+		:tag('td'):css('text-align','center'):css('font-weight','bold')
+			:tag('span'):css('color','#000'):css('background-color',Color['stay']):css('border','none')
+				:css('border-radius','0px'):css('letter-spacing','0.1em')
+				:css('font-family','\'Source Code Pro\',monospace')
+				:wikitext('DECIDER'):done()
+			:done()
```

Comment:
outsource lots of this to css

Reply:
Moved to css and general cleanup of teh code